tests/PhpSpreadsheetTests/Style/AlignmentTest.php

```diff
@@ -84,6 +84,17 @@ public function testHorizontal(): void
         self::assertEquals(0, $cell3->getStyle()->getAlignment()->getIndent());
     }
 
+    public function testJustifyLastLine(): void
+    {
+        $this->spreadsheet = new Spreadsheet();
+        $sheet = $this->spreadsheet->getActiveSheet();
+        $cell1 = $sheet->getCell('A1');
+        $cell1->setValue('ABC');
+        $cell1->getStyle()->getAlignment()->setHorizontal(Alignment::HORIZONTAL_DISTRIBUTED);
```

Comment:
Your test is not doing enough (or you want to add an additional test in Reader/Xlsx). We want to write and then read the spreadsheet to prove that both Write and Read handle the new attribute correctly. Look for examples of this by searching for "writeAndReload" in the PhpSpreadsheetTests Reader/Xlsx directory.

Reply:
Resolved in latest push.